src/components/SearchBox/SearchBox.tsx

```diff
@@ -67,6 +67,7 @@ const SearchBox = React.forwardRef<HTMLInputElement, Props>(
       onChange && onChange("");
       if (internalRef.current) {
         internalRef.current.value = "";
+        forwardedRef.focus();
```

Comment:
I believe it should be:

```suggestion
        internalRef.current.focus();
```